qemu/tests/cfg/virtual_nic.cfg

```diff
@@ -5,6 +5,7 @@
         - private:
             type = virtual_nic_private
             vms = "vm1 vm2 vm3"
+            mem = 4096
```

Comment:
`mem` in test case will be overwritten by the command line, please use `mem_fixed`, thanks!

Reply:
I have changed it, please help me review it again.Thanks.